### PR TITLE
Update/fix mounts for comfyUI

### PR DESCRIPTION
**Pull request by LEv145**
I updated the paths to make all paths work

Now paths in ComfyUI are working
Added: 
- `input`
- `models/configs`
- `models/gligen`
- `models/diffusers`
- `models/hypernetworks`

Was broken:
- `custom_nodes`
- `models/clip_vision` 
- `models/clip`

### CODE REVIEW

File: services/comfy/entrypoint.sh [hunk 1]
```diff
@@ -8,26 +8,32 @@ declare -A MOUNTS
...
-MOUNTS["${ROOT}/models/custom_nodes"]="/data/config/comfy/custom_nodes"
-MOUNTS["${ROOT}/models/style_models"]="/data/config/comfy/style_models"
-MOUNTS["${ROOT}/models/t2i_adapter"]="/data/config/comfy/t2i_adapter"
+MOUNTS["${ROOT}/input"]="/data/config/comfy/input"
+MOUNTS["${ROOT}/custom_nodes"]="/data/config/comfy/custom_nodes"
+MOUNTS["${ROOT}/models/configs"]="/data/config/comfy/configs"
+
+MOUNTS["${ROOT}/models/style_models"]="/data/config/comfy/models/style_models"
+MOUNTS["${ROOT}/models/t2i_adapter"]="/data/config/comfy/models/t2i_adapter"
+MOUNTS["${ROOT}/models/clip"]="/data/config/comfy/models/clip"
```
Comment (AbdBarho) [inline services/comfy/entrypoint.sh:33]: is this any different from the standard clip model provided by the transformers packages?

File: services/comfy/entrypoint.sh [hunk 2]
```diff
@@ -8,26 +8,32 @@ declare -A MOUNTS
...
+MOUNTS["${ROOT}/models/clip_vision"]="/data/config/comfy/models/clip_vision"
+MOUNTS["${ROOT}/models/gligen"]="/data/config/comfy/models/gligen"
+MOUNTS["${ROOT}/models/diffusers"]="/data/config/comfy/models/diffusers"
```
Comment (AbdBarho) [inline services/comfy/entrypoint.sh:36]: same here?

**Review by AbdBarho — COMMENTED** [review]
Thanks, we still need some tuning for this UI.

**Comment by LEv145** [comment]
I didn't really understand you, but comfyUI has nodes that use these models: 
- clip_vision([clip vision models](https://github.com/comfyanonymous/ComfyUI/blob/master/models/clip_vision/put_clip_vision_models_here)): [`CLIPVisionLoader`](https://github.com/comfyanonymous/ComfyUI/blob/master/nodes.py#L462)
- gligen([gligen models](https://github.com/comfyanonymous/ComfyUI/blob/master/models/gligen/put_gligen_models_here)): [`GLIGENLoader`](https://github.com/comfyanonymous/ComfyUI/blob/master/nodes.py#L554)
- clip([clip or text encoder models](https://github.com/comfyanonymous/ComfyUI/blob/master/models/clip/put_clip_or_text_encoder_models_here)): [`CLIPLoader`](https://github.com/comfyanonymous/ComfyUI/blob/master/nodes.py#L447)
- diffusers([diffusers models](https://github.com/comfyanonymous/ComfyUI/blob/master/models/diffusers/put_diffusers_models_here)): [`DiffusersLoader`](https://github.com/comfyanonymous/ComfyUI/blob/master/nodes.py#L284)

**Comment by LEv145** [comment]
@AbdBarho Did I make myself clear?

File: services/comfy/entrypoint.sh (hunk 1 above)
Comment (LEv145) [reply services/comfy/entrypoint.sh:33]: I think yes

**Comment by AbdBarho** [comment]
I will try to test the clip model and others tomorrow, then we can merge this PR, sorry for the delay.

**Comment by PassiveLemon** [comment]
What was broken about these? I never had to use them so I don't really know

**Comment by AbdBarho** [comment]
I changed the solution to something a bit more maintainable.

The commented lines are stuff I was not able to test, so I will delay for now. If you have examples of how I can test this stuff, hmu.

**Review by AbdBarho — APPROVED** [review]

**Comment by LEv145** [comment]
@PassiveLemon It was impossible to add custom nodes, or for example load your own `clip` for `CLIPLoader` node for `CLIPTextEncode`:
https://github.com/comfyanonymous/ComfyUI/issues/25#issuecomment-1442881979

**Comment by LEv145** [comment]
@AbdBarho 
You can also use this example for tests `clip`: 
https://github.com/comfyanonymous/ComfyUI/issues/25#issuecomment-1442881979

**Comment by AbdBarho** [comment]
Thank you! I will try this example and see what need to be changed.

**Comment by PassiveLemon** [comment]
> @PassiveLemon It was impossible to add custom nodes, or for example load your own `clip` for `CLIPLoader` node for `CLIPTextEncode`: [comfyanonymous/ComfyUI#25 (comment)](https://github.com/comfyanonymous/ComfyUI/issues/25#issuecomment-1442881979)

This actually means a lot to me. I was just working on a way to make nodes easier to install by automatically installing their requirements.txt when put into the custom_nodes directory. This would explain why it wasn't working, because now it does work lol

**Comment by AbdBarho** [comment]
So the `clip` is just replacement for the clip that is usually part of the sd model, I see.

I created #458 for clip, I am unfortunately still unsure about the remaining stuff, I will checkout #451 next

**Pull request merged**